### ISSUE TITLE
How to connect to Windows 10

### ISSUE THREAD

**Issue by oliverhildner** [OverallLine5/MobileBandSync#17]
Hello, thank you for your amazing work! I really apprecriate it. 
I got the app installed and running but it seems like I can't find a Way to connect my Band 2.

First i tried USB, then Bluetooth. 
So I paired the band with my PC by entering a Pairing Code and all. Unfortunately the app always says "Nicht verbunden", no matter what I try:
![grafik](https://github.com/OverallLine5/MobileBandSync/assets/50845166/d911a499-317b-4ecd-95f7-d35d3e48d294)

The USB Device as well as the Bluetooth Device are visible in my Bluetooth settings. 
![grafik](https://github.com/OverallLine5/MobileBandSync/assets/50845166/e1f0f09b-b878-4859-9a6a-6694e364ea8f)
I tried both of them connected or only one of them connnected. 

Am I doing it wrong? Whats the intended way to connect the band?

**Comment by OverallLine5** [comment]
Thanks for testing, I will have a look at it next week. Until then, you can try the following: Remove the paired device you don't need ("Microsoft Band 2"), switch off Bluetooth for "MSFT Band 2". Close DesktopBandSync. Switch on Bluetooth again and start DesktopBandSync. I think it's a Bluetooth LE issue.

**Comment by oliverhildner** [comment]
Thank you. 
This didn't work for me so far. 
I think it's also worth noticing that after pairing the device it says "connected" for some seconds just before switching back to "paired" or "gekoppelt" on my PC.
I did try restarting and reinstallling my bluetooth services so far but with no successs.
Altough my PC and Band know about each other (the Band is listed under "Weitere Geräte" in my Bluetooth-Settings and my PC is listed as "Gekoppelt mit" in my bands info page) the devices don't seem to be connected at all. 
Windows doesn't act like that and also the band doesn't have the bluetooth-indicator-icon turned on when swiping tho the outer left side of the screen where the indidactor-icons for bluetooth, heart-rate and battery are.

**Comment by OverallLine5** [comment]
![connect](https://github.com/OverallLine5/MobileBandSync/assets/69265068/ba475c67-cf7a-4119-93ae-dfec824d13c9)
Hi Oliver,

I just tested it on my Win11 machine (Surface Pro 8) and everything works as expected. You should see your Band 2 device in the list of BT devices. The state must be 'connected', otherwise DesktopBandSync cannot find your Band 2. Have you ever established a working BT connection with any device? My guess is that BT does not work. Unfortunately, DesktopBandSync does not work with a USB connection.

Regards,
Marcus

**Comment by oliverhildner** [comment]
Thank you for your answer.

Knowing this and the fact that Bluetooth on my PC was working just fine I came up with a solution that worked for me.
I was now suspecting my band to be the reason for this issue. This is how i got it working:

- Factory reset the band and revive it with the amazing [Band unlocker](https://bandunlocker.briosche.de/) (Just follow instructions there)
- When being prompted to choose a phone to pair with, I went for "Android" but paired with my PC instead
- Afterward let Band unlocker do it's thing and after the band is set up, enable bluetooth on the band
- It should automatically reconnect to the PC

Now the band is connected to the app and I am looking very much forward to trying it out  :)

Thank you again!
